#! /usr/bin/env python
from dynamic_reconfigure.parameter_generator_catkin import *

PACKAGE = "fast_lio"
gen = ParameterGenerator()
gen.add("cutting_method",bool_t,0,"default setting is to crop the top and bottom separately",True)
gen.add("z_", double_t, 0, "z_up and down adjustment", 0.0, -20.0, 20.0)
gen.add("z_high", double_t, 0, "rebuild the z high", 1.0, -20.0, 20.0)
gen.add("z_low", double_t, 0, "rebuild the z low", -1.0, -20.0, 20.0)
gen.add("leaf_size", double_t, 0, "fliter_registered_PC", 0.5, 0.1, 20.0)
gen.add("leaf_size_filter_accMap", double_t, 0, "fliter_acc_PC", 0.2, 0.1, 20.0)

exit(gen.generate(PACKAGE, "Point_cloud_visualization", "debug_para"))
